pom.xml

```diff
@@ -313,10 +314,6 @@
             <version>3.1.0</version>
             <scope>provided</scope>
         </dependency>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-actuator</artifactId>

```

Comment:
why remove actuator?

Reply:
actuator is present twice, which is why I decided to do this.